geojson/src/commonMain/kotlin/org/maplibre/spatialk/geojson/Object.kt

```diff
@@ -6,7 +6,7 @@ package org.maplibre.spatialk.geojson
  *
  * @property bbox An optional bounding box used to represent the limits of the object's geometry.
  */
-public sealed interface GeoJson {
+public sealed interface Object {
```

Comment:
the name here makes me think of a java `Object` (equivalent of Kotlin `Any`)

imo this should be `GeoJsonObject`, matching [the spec](https://datatracker.ietf.org/doc/html/rfc7946#section-3).

Reply:
Done